calc.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <title>Mac Calculator Clone</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+  <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <div class="calculator-container">
+    <div class="calculator-window">
+      <div class="calculator-display">
+        <div class="calculator-history"></div>
+        <div class="calculator-current-input"></div>
+      </div>
+      <div class="calculator-control">
+        <div class="calculator-row">
+         <div class="calculator-field"><button>C</button></div><!--
+           --><div class="calculator-field"><button>+/-</button></div><!--
+           --><div class="calculator-field"><button>%</button></div><!--
```

Comment:
I'm not sure what the standard is for using things other than <button> elements for this, but you don't need an element wrapped in a div for the calculator buttons.  (I just used a straight-up div, myself).  This will make some of your CSS and JS easier.